README.md

```diff
@@ -94,14 +89,14 @@ After a [discussion](https://github.com/OpenZWave/Zwave2Mqtt/issues/201) with Op
 
 ```bash
 # Using volumes as persistence
-docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 --mount source=zwave2mqtt,target=/usr/src/app/store robertslando/zwave2mqtt:latest
+docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store robertslando/zwavejs2mqtt:latest
 
 # Using local folder as persistence
 mkdir store
-docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store robertslando/zwave2mqtt:latest
+docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store robertslando/zwavejs2mqtt:latest
```

Comment:
Can we setup a dockerhub org. Also probably worth pushing to GitHub's new registry given the incoming dockerhub annoying request throttling

Reply:
I thought the same! @AlCalzone do you want me to setup it? Do you have a docker account?

Reply:
You can set it up. I think I have a docker account, but I need to look it up

Reply:
Done, will ad you and chrisns ad owners too, is your username the same as github?

Reply:
I have found `alcalzone`

Reply:
Let me know @chrisns and @AlCalzone if you received the invitation

Reply:
Yup 👍

Reply:
got it

Reply:
Perfect